cdap-ui/templates/ETLRealtime/Table.json

```diff
@@ -24,6 +24,16 @@
              "widget": "textbox",
              "label" : "Row Field",
              "description" : "Specify the field from the record above that should be used as key"
+          },
+
+          "case.sensitive.row.field" : {
```

Comment:
There is no file ETLBatch/Table.json, think it uses common/Table.json which is why I put it in there